0001-pip-anti-gaming.md

```diff
@@ -0,0 +1,103 @@
+* It should be pointed out that gamers have worked with Pollen team after the fact to inform them about this issue and how to discover this in system logs. 
+
+#### Pollen addressed this exploits in phases:
+ 1. BB Anti Swarming (Rewards split if 2 or more BB's validate a hex at the same time).
+ 2. Max hex validation per flower set to fixed number regardless if BB's validate more unique hexes (D = 7, C = 7, E = 10 and BC = 20). Flower and BB's still get 2 pics each for each unique hex however up to max for flower per BB.
+
+#### What could further be done. 
+1. Reduce max Hex for a Camellia as its the most commonly gamed. 
+ * Motivation: Its an indoor flower, a hex is several city block, and its therefore not possible for a Camellia to cover more then 1 hex.
+ *  Side effects: A Camellia could be place in an outdoor enclosure on balcony, roof etc plus this would effect C owners that never gamed the rewards as well.
+2. Use signal strength FSPL to filter out impossibles validation location.
+3. Require certain number of GPS satellite looks before validation is possible to minimize drift.
+
+### The moving Flower
+* Deliberate exploit by driving around with a flower (C or D), powered by battery and tethered internet connection. 
+* Flower and BBs could be in the same vehicle and new hexes could be validated while driving to new hexes. This scheme resulted in the same large rewards as the GPS drift exploit. Early on many BBs could validate hexes at the same time, flowers got 10 pics per BB and hex etc. 
+
+#### Pollen addressed this exploits in phases (Same fixes as for GPS drift):
+ 1. BB Anti Swarming (Rewards split if 2 or more BB's validate a hex at the same time).
+ 2. Max hex validation per flower set to fixed number regardless if BB's validate more unique hexes (D = 7, C = 7, E = 10 and BC = 20). Flower and BB's still get 2 pics each for each unique hex however up to max for flower per BB.
+
+#### What could further be done. 
+1. If a flower changes GPS location the flower wouldnt get any rewards for that day. 
+ * Motivation: Moving a flower would have a fee (similar to Helium) but the fee would be a loss of rewards not something flower owner have to pay.
+ *  Side effects: Possible GPS drift could cause rewards loss. Could be mitagted in post process of data.
```

Comment:
This should be dug into and tested before implementation. Some of the best implementations of hotspots are also going to be high up so we really need to make sure we're not dinging those.